static/app/views/performance/browser/resources/shared/resourceSize.tsx

```diff
@@ -0,0 +1,19 @@
+import {Fragment} from 'react';
+
+import FileSize from 'sentry/components/fileSize';
+
+type Props = {
+  bytes: number;
+};
+
+function ResourceSize(props: Props) {
+  const {bytes} = props;
+  const hasNoSize = bytes === 0;
+  if (hasNoSize) {
```

Comment:
super nit: I think the introduction of another variable is not needed, the boolean check is plenty self-descriptive.

```suggestion
  if (bytes === 0) {
```

Reply:
no worries, I agree!